elastic/kibana
# [ResponseOps][flapping] change action behavior when flapping

Resolves https://github.com/elastic/kibana/issues/143445

## Summary

This pr changes the behavior of alerting actions respecting flapping. 
To recover a flapping alert the alert must be reported as recovered for 4 (this will be configurable eventually) consecutive runs before the framework makes the alert as recovered.

Link to a spreadsheet that helps vizualize the changes: https://docs.google.com/spreadsheets/d/1amoUDFx-Sdxirnabd_E4OSnyur57sLhY6Ce79oDYgJA/edit#gid=0

Also Resolves https://github.com/elastic/kibana/issues/147205 to recover those alerts 


### Checklist


### To verify
- Create an alert and let it run until it gets into a flapping state (changes from active/recovered at least 4 times)
- Once flapping, verify the following cases:
- Verify that a recovered alert will be returned as an active notification 3 times before finally recovering
- Verify that an alert that does not recover 3 times before an active alert is reported never reports as a recovered alert
